hack/fill-empty-docs.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+GOPATH="${GOPATH:-~/go}"
+export GOFLAGS="${GOFLAGS:-"-mod=vendor"}"
+
+export PATH=$PATH:$GOPATH/bin
+export SUITES_PATH=cnf-tests/bin
+
+# generate the junit files
+hack/build-test-bin.sh
+cnf-tests/test-run.sh -junit _cache/junit -ginkgo.dryRun -ginkgo.v
+
+go build -o _cache/docgen cnf-tests/docgen/main.go 
+# use the junit files to fill the descriptions
+_cache/docgen fill --junit _cache/junit/cnftests-junit.xml --description cnf-tests/docgen/e2e.json
+_cache/docgen fill --junit _cache/junit/validation_junit.xml --description cnf-tests/docgen/validation.json
```

Comment:
nit: why this file is `_` and the other one is `-`?

Reply:
It ended up being like this, not changed in this PR. That could be made better.